### PR TITLE
Archive Radio Galaxy Zoo

### LINKED ISSUE #19: Create 'interim' page for Radio Galaxy Zoo
Please create an 'interim' page following the whaling.oldweather.org/#/ model. 

(Please remove the 'old weather' button). 

Below is the text and attached is the image to use. 

Radio Galaxy Zoo

As supermassive black holes in the centers of galaxies consume their surrounding material, large amounts of radio synchrotron emission is released in the form of radio jets.  These galaxies are called radio galaxies. Through Radio Galaxy Zoo, citizen scientists have helped assemble one of the largest catalogues of radio galaxy classifications and discovered many more radio galaxies with unusual morphologies. The picture on the left shows one of the earliest discoveries made by Radio Galaxy Zoo citizen scientists, a giant wide-angle tailed galaxy which in turn revealed a previously uncatalogued cluster of galaxies (named the Matorny-Terentev cluster after the citizen scientists who first made this discovery; Banfield et al 2016).

(With the words 'Banfield et al 2016' linked to http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.460.2376B&link_type=ARTICLE&db_key=AST&high= 

gwat

### PR DESCRIPTION
**Pull request by wgranger**
Creates the archived page for Radio Galaxy Zoo. This PR contains a standalone landing page.

Closes #19

### CODE REVIEW

File: RadioGalaxyZoo/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
```
Comment (eatyourgreens) [inline RadioGalaxyZoo/index.html]: Minor point but this should be `lang="en"`.

File: RadioGalaxyZoo/index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Radio Galaxy Zoo: Archive</title>
+        <meta name="description" content="">
```
Comment (eatyourgreens) [inline RadioGalaxyZoo/index.html]: Do we want a description here for search engines?

**Review by eatyourgreens — APPROVED** [review]

File: RadioGalaxyZoo/index.html (hunk 2 above)
Comment (wgranger) [reply RadioGalaxyZoo/index.html]: Yes, I can add something in here. Probably something general like "Zooniverse Radio Galaxy Zoo: Archived Page"

**Comment by wgranger** [comment]
I've made changes to the metadata. This should be ready to go.

**Pull request merged**